src/auspex/pulse_calibration.py

```diff
@@ -298,13 +300,174 @@ def calibrate(self):
 
         print("DRAG", opt_drag)
 
-        with open(config.channelLibFile, 'r') as FID:
-            chan_settings = json.load(FID)
-        chan_settings['channelDict'][self.qubit_name]['pulseParams']['dragScaling'] = fitted_drag
-        self.update_libraries([chan_settings], [config.channelLibFile])
+        self.chan_settings['channelDict'][self.qubit_names[0]]['pulseParams']['dragScaling'] = fitted_drag
+        self.update_libraries([self.chan_settings], [config.channelLibFile])
 
         return fitted_drag
 
+'''Two-qubit gate calibrations'''
+class CRLenCalibration(PulseCalibration):
```

Comment:
It seems there is a lot of repeated code between the `CRLen`, `CRPhase` and `CRAmp`. It may not be worth the trouble but I wonder if a `CRCalibration` class could keep it DRY